### DISCUSSION TITLE
Did anyone used this on parallel desktop M1?

### DISCUSSION THREAD

**Discussion by urfdvw** [valinet/ExplorerPatcher#701] [Q&A] [answered]
M1 Mac PD App store version. Win 11 Education. Nothing happened after running the app.

**Comment by matzoshaina** [comment, marked as answer]
I use parallels access for windows. It is not compatible with explorepatcher.  The only way to run this application is to temporarily uninstall EP. I suspect this incompatibility extends to all parallels products

**Reply by urfdvw** [reply]
Thanks!

**Comment by valinet** [comment]
I don't understand why it is implied that this must work on Apple M1 processors (or ARM64-based processors in general). ExplorerPatcher only works on amd64 at the moment, as clearly stated here: https://github.com/valinet/ExplorerPatcher/wiki/Reporting-problems. There are numerous threads about this already, if that's what this thread is all about.

**Reply by urfdvw** [reply]
Oh yep. Thanks, I did search in this in discussions but found nothing. That's why I ask. As a user, it is not natural for me to go to Wiki if I read the whole read me, found Issue is disabled? and searched in the Discussion.

**Reply by valinet** [reply]
Results for a simple search for "arm" (some match "m1" as well) in issues and discussions:

https://github.com/valinet/ExplorerPatcher/issues/689
https://github.com/valinet/ExplorerPatcher/issues/617
https://github.com/valinet/ExplorerPatcher/issues/136
https://github.com/valinet/ExplorerPatcher/discussions/566

The README strongly advises for checking out previous threads and the documentation (wiki) for good reasons. It should be natural to read the manual before operating a device.

**Reply by urfdvw** [reply]
The first thing I saw in **Issues** is "DO NOT FILE ISSUES", which makes me think you guys don't like issues and disabled it. I searched "parallel desktop" in the **discussions** and nothing. If you do it now will find that this is the only post.

**Reply by urfdvw** [reply]
You know. I am not complaining at all. I just could not find the information in the usual ways, so I asked. I appreciate all your work and sharing, but you do need to give better directions instead of forcing less informed people like me to make the same mistake again.

**Reply by valinet** [reply]
> The first thing I saw in Issues is "DO NOT FILE ISSUES", which makes me think you guys don't like issues and disabled it.

guy*

Also, yeah, I don't like issues, but if they happen, they get a ticket there. I don't like people opening tickets there on their own either, as that is the place to track actual issues, and indeed, the place for other stuff is Discussions, as explained [here](https://github.com/valinet/ExplorerPatcher/wiki/Reporting-problems).

> I searched "parallel desktop" in the discussions and nothing. If you do it now will find that this is the only post.

Because "parallel desktop" is not relevant for the topic here. "M1" or "ARM64" are more relevant.

> but you do need to give better directions instead of forcing less informed people like me to make the same mistake again

I am not forcing you to do anything.

I could add a mention in the README that this is amd64-only at the moment.
